sources/assets/Stride.Core.Assets/SolutionPlatform.cs

```diff
@@ -276,14 +267,13 @@ public SolutionConfiguration(string name)
         {
             if (name == null) throw new ArgumentNullException("name");
```

Comment:
as youve added later on also nameof it would be nice to have here also a nameof `throw new ArgumentNullException(nameof(name))`